src/Microsoft.AspNet.Mvc.Core/ParameterBinding/ModelBindingHelper.cs

```diff
@@ -132,12 +136,13 @@ public static class ModelBindingHelper
                [NotNull] IModelMetadataProvider metadataProvider,
                [NotNull] IModelBinder modelBinder,
                [NotNull] IValueProvider valueProvider,
+               [NotNull] IObjectModelValidator objectModelValidator,
```

Comment:
Might want to update the description of this method to say this method specifically validates the model in addition to binding it.